File: VB/E2836/Controllers/HomeController.vb
Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        ViewData("Message") = "How to bind GridView to XPO in Server Mode"

        Return View()
    End Function

    Public Function GridViewPartial() As ActionResult
        Return PartialView("_GridViewPartial")
    End Function
End Class

File: CS/E2836/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace E2836.Controllers
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            ViewData["Message"] = "How to bind GridView to XPO in Server Mode";

            return View();
        }

        public ActionResult GridViewPartial()
        {
            return PartialView("_GridViewPartial");
        }
    }
}